Misc/NEWS.d/next/Tests/2018-07-31-23-33-06.bpo-33613.Cdnt0i.rst

```diff
@@ -0,0 +1,5 @@
+Fix a race condition in ``multiprocessing.semaphore_tracker`` when the
+tracker receives SIGINT before it can register signal handlers for ignoring
+it. Improve ``_test_multiprocessing.test_semaphore_tracker_sigint`` and
```

Comment:
_test_multiprocessing is a private module. Usually I just write "multiprocessing tests". For example:

Improve test_semaphore_tracker_sigint() of multiprocessing tests.